### ISSUE TITLE
2.3.1 - Sorting by Score column when editing scores does not work correctly

### ISSUE THREAD

**Issue by grmelacz** [geoffhumphrey/brewcompetitiononlineentry#1258]
Version: 2.3.1

Installation URL: https://soutez.mhmeeting.cz

Is your installation hosted on brewcompetition.com or brewcomp.com? No

Description of Issue: Those are 2 interconnected issues:
1. When I enter a new value to the Score column of the scores table list and then click the column header, the sorting does not work correctly - the new values, no matter if lower or higher are not taken in consideration when sorting.
2. There is also a problem with sorting values lower than 10 as they're put before the highest values in the column (see screenshot). Note: Maybe only sometimes? Now, when the competition is finished, it appears to sort it correctly - I suspect the problem to exist only when some values are not filled in yet.

<img width="1283" alt="Screenshot 2021-12-04 at 17 09 26" src="https://user-images.githubusercontent.com/41264018/144889502-9d514821-c94a-448f-991b-fd0d58c57e39.png">

**Comment by geoffhumphrey** [comment]
Hi @grmelacz. Confirmed. A fix is included in the ab443aa0 commit and will be included in the next release.

**Comment by geoffhumphrey** [comment]
A testing js source was left in the last commit that may cause errors. This commit fixes that issue - 13527656. Or, simply change line 6 in judging_scores.admin.php back to:
```
<script src="<?php echo $base_url;?>js_includes/admin_ajax.min.js"></script>
```

**Comment by grmelacz** [comment]
Hey @geoffhumphrey, thanks for the fix! That is great news.
